detect_variants/exac_filter.cwl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "ExAC filter"
+baseCommand: ["/usr/bin/perl", "/opt/vep/ensembl-vep/filter_vep.pl"]
+requirements:
+    - class: ShellCommandRequirement
```

Comment:
Would this work without the `ShellCommandRequirement`?

Reply:
That is a good question. I'm not sure.